### DISCUSSION TITLE
Configuration of  kubeflow deployment

### DISCUSSION THREAD

**Discussion by michael-gvdw** [deployKF/deployKF#194] [Questions & Answers]
Hello everyone, 
I am currnetly trying to connect to an external object store. I have followed the local deployment guide and altered the sample-values.yaml file (locally). Unfortunatelly the cofiguration values do not seem to be applied. What could i be doing wrong here?

**Comment by thesuperzapper** [comment]
@michael-gvdw please make sure you are following the ["connect external object store"](https://www.deploykf.org/guides/external/object-store/#connect-an-external-object-store) guide.

Or if you need help understanding how values work, see [this page](https://www.deploykf.org/guides/values/).
